ironic-images/ipa-debian-bookworm/ipa-debian-bookworm.yaml

```diff
@@ -0,0 +1,8 @@
+- imagename: ipa-debian-bookworm
+  elements:
+  - ironic-python-agent-ramdisk
+  - debian
+  - devuser
+  - simple-init
```

Comment:
```suggestion
```

Don't think we need simple-init cause IPA is gonna force some kind of init on us.

Reply:
Updated